packages/financial-templates-lib/test/truffle/UniswapPriceFeed.js

```diff
@@ -418,13 +469,13 @@ contract("UniswapPriceFeed.js", function(accounts) {
       // The historical TWAP for 1 hour ago (the earliest allowed query) should be 100 for the first half and then 90 for the second half -> 95.
       assert.equal(
         (await scaleUpPriceFeed.getHistoricalPrice(currentTime - 3600)).toString(),
-        toBN(toWei("95"))
+        toBN(parseFixed("95", baseCurrencyPrecision).toString())
```

Comment:
do you want to create a `convertPrice` or `convertBaseCurrency` method like we do in other places that basically will wrap `parseFixed`?

Reply:
Done